contrib/nydusify/cmd/nydusify.go

```diff
@@ -364,7 +364,12 @@ func main() {
 					Usage:   "Read prefetch list from STDIN, please input absolute paths line by line",
 					EnvVars: []string{"PREFETCH_PATTERNS"},
 				},
-
+				&cli.StringFlag{
+					Name:    "compressor",
```

Comment:
Thanks for adding this, please also add the option for `nydusify pack` subcommand.

Reply:
fixed